Mathlib/Analysis/Normed/Group/Constructions.lean

```diff
@@ -6,6 +6,8 @@ Authors: Patrick Massot, Johannes Hölzl, Yaël Dillies
 import Mathlib.Algebra.Group.ULift
 import Mathlib.Algebra.PUnitInstances.Algebra
 import Mathlib.Analysis.Normed.Group.Basic
+import Mathlib.Topology.Algebra.SeparationQuotient
+import Mathlib.Analysis.Normed.Group.Uniform
```

Comment:
Without this import, Lean does not know that normed groups are topological groups.